.github/workflows/vulnerability_scanning.yml

```diff
@@ -2,45 +2,24 @@ name: Workflow to scan vulnerability
 on:
   workflow_call:
     inputs:
-      image_tar_name:
-        type: string
-        required: false
+      docker_tag:
+          type: string
+          required: true
       runner:
         ## ['cn', 'us', 'cn1', 'cn2', 'cn3', 'us1', 'us2', 'us3', ' ubuntu-latest']
         type: string
-        ## use github hosted runner by default
+        # ## use github hosted runner by default
         default: "ubuntu-latest"
         required: false
-    secrets:
-      MINIO_ACCESS_KEY_ID:
-        required: false
-      MINIO_SECRET_ACCESS_KEY:
-        required: false
-      MINIO_ENDPOINT_URL:
-        required: false
+
+
 jobs:
   scanning:
     name: vulnerability scanning
-    runs-on: ${{ inputs.runner }}
+    runs-on: [ self-hosted, "${{ inputs.runner }}" ]
```

Comment:
no need self-hosted label in here

Reply:
I think must have `self-hosted`, or it doesn't support runner variables here.

Reply:
I don't think do.  as we use  `runs-on: ${{ inputs.runner }}`  before which worked well.